doc/tutorial.rst

```diff
@@ -119,8 +119,42 @@ Evaluate the energy
 A rough estimate of the expectation value of the energy of a trained wave function can be obtained already from the local energies of the training run. A rigorous estimation of the energy expectation value up to the statistical sampling error can be obtained when evaluating the energy expectation value of the trained wavefunction without further optimization. This is achieved by passing a training checkpoint is passed to the :func:`~deepqmc.train` function, but the optimizer is specified to be ``None``:
 
     >>> import jax.numpy as jnp
-    >>> train_state = jnp.load('workdir/training/chkpt-10000.pt',allow_pickle=True)
-    >>> train(H, net, None, sampler, train_state=train_state, steps=500, sample_size=2000, seed=42)
+    >>> step, train_state = jnp.load('runs/01/training/chkpt-10000.pt',allow_pickle=True)
+    >>> train(H, ansatz, None, sampler, train_state=train_state, steps=500, sample_size=2000, seed=42)
     evaluating: 100%|█████████| 500/500 [01:20<00:00,  6.20it/s, E=-8.07000(19)]
 
 The evaluation generates the same type of logs as the training, but writes to ``workdir/evaluation`` instead. The final energy can be read from the progress bar, the Tensorboard event file or computed from the local enregies in the hdf5 file respectively.
+
+Pseudopotentials
+-------------------
+
+DeepQMC currently supports ``bfd`` [Burkatzki07]_ and ``ccECP`` [Bennett17]_ pseudopotentials, which can be enabled by passing the ``pp_type`` argument to the molecule definition. This replaces a certain number of core electrons with a pseudopotential, reducing the total number of electrons explicitly treated and thus decreasing the computational cost. If the argument ``pp_type`` is passed, the pseudopotentials are used for all the nuclei in the molecule considered, but can be turned off for individual nuclei by specifying ``pp_mask``, a boolean array with ``True`` for each nucleus with pseudopotential turned on. The following example defines a TiO molecule where the titanium core is replaced by a pseudopotential and the oxygen core is left unaffected::
+
+    mol = Molecule(  # TiO
+        coords=[[0.0, 0.0, 0.0], [1.668, 0.0, 0.0]],
+        charges=[22, 8],
+        charge=0,
+        spin=2,
+        unit='angstrom',
+        pp_type='ccECP',
+        pp_mask=[True,False],
+    )
+
+The systems containing heavier atoms sometimes tend to produce NaN errors. To avoid these issues, it was found useful to use a smaller initial variance of electrons (via the ``elec_std`` argument) and a larger decorrelation length for sampling::
```

Comment:
```suggestion
The systems containing heavier atoms sometimes tend to produce NaN errors. To avoid these issues, it was found useful to use a smaller variance for the initial distribution of electrons around the nuclei (via the ``elec_std`` argument) and a larger decorrelation length for sampling::
```

Reply:
I think we should make it so that this is no longer needed, but this would be another PR.